regenmaschine/endpoints/watering.py

```diff
@@ -30,6 +30,20 @@ async def log(
         data = await self.controller.request("get", endpoint)
         return cast(List[Dict[str, Any]], data["waterLog"]["days"])
 
+    @EndpointManager.raise_on_gen1_controller
+    async def flowmeter(self) -> dict[str, Any]:
+        """Return the registered values from flowmeter."""
+        data = await self.controller.request("get", "watering/flowmeter")
+        return data
+
+    @EndpointManager.raise_on_gen1_controller
+    async def post_flowmeter(self, value: float, units: str = "litre") -> int:
+        """Push flowmeter values to unit."""
+        data = await self.controller.request(
+            "post", "watering/flowmeter", json={"value": value, "units": units}
+        )
+        return cast(int, data["statusCode"])
```

Comment:
Let's return the entire response data (since that's consistent with what we do elsewhere).

Reply:
Updated.